src/pool/srv_iv.c

```diff
@@ -549,7 +549,7 @@ pool_iv_update(void *ns, int class_id, struct pool_iv_entry *pool_iv,
 	pool_key = (struct pool_iv_key *)key.key_buf;
 	pool_key->pik_entry_size = pool_iv_len;
 	rc = ds_iv_update(ns, &key, &sgl, shortcut, sync_mode, 0,
-			  true /* retry */);
+			  false /* retry */);
```

Comment:
so all pool iv update will without automatic retry? (now it also used for prop_update).
Or can add a retry parameter to pool_iv_update() and ds_pool_iv_map_update()?

Reply:
The user suppose to handle pool prop iv distribute outside pool_iv_update(), IMHO, otherwise it is easy to cause some strange endless loop here. 

Specially in this case, I think the user suppose to update the pool map then update the pool prop, so it should not met GRPVER here IMHO.